<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAxes</class>
 <widget class="QDialog" name="DialogAxes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Axes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutX">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutXmin">
       <item>
        <widget class="QLabel" name="labelXmin">
         <property name="text">
          <string>x min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxXmin">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutXmax">
       <item>
        <widget class="QLabel" name="labelXmax">
         <property name="text">
          <string>x max</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxXmax">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutY">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutYmin">
       <item>
        <widget class="QLabel" name="labelYmin">
         <property name="text">
          <string>y min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxYmin">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutYmax">
       <item>
        <widget class="QLabel" name="labelYmax">
         <property name="text">
          <string>y max</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxYmax">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMirrorX">
     <item>
      <widget class="QLabel" name="labelMirrorX">
       <property name="text">
        <string>Mirror X-Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMirrorX">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMirrorY">
     <item>
      <widget class="QLabel" name="labelMirrorY">
       <property name="text">
        <string>Mirror Y-Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMirrorY">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAxes</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAxes</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
